base/install_opcua.sh

```diff
@@ -3,6 +3,8 @@
 set -ex
 
 . /opt/epics/install-functions.sh
+. /opt/epics/epics_versions.sh
```

Comment:
`$EPICS_BASE_PATH/epics_versions.sh`?

Reply:
We don't use `$EPICS_BASE_PATH` to source `install-functions.sh`, though. Should we do that? Otherwise it doesn't seem very consistent.